copilot/webapp/manifest.yml

```diff
@@ -29,10 +29,9 @@ exec: true # Enable running commands in your container.
 network:
   connect: true # Enable Service Connect for intra-environment traffic between services.
   vpc:
-    flow_logs: on
-    deny_default: true
     security_groups:
-      - from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-dbSecurityGroup
+      groups:
+        - from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-dbSecurityGroup
```

Comment:
Do we know that this works? I know that's how we've understood the docs, but just wondering if this is tested vs what we had before without `groups:`

Reply:
The app does boot successfully, so I assume so...